dist/loglevel.min.js

```diff
@@ -1,3 +1,3 @@
 /*! loglevel - v1.9.2 - https://github.com/pimterry/loglevel - (c) 2024 Tim Perry - licensed MIT */
```

Comment:
Doesn't really matter on a branch like this, but up until now I've only been updating the `dist/` files at the point of actual releases, and other changes have just been applied to `lib/` by itself. There's no strong case for it though beyond convention I guess.

In fact, the only reason `dist` is really committed at all is because we used to use github as a CDN, but nowadays unpkg can easily serve the official release content out of the npm package, so we could git-ignore `dist` entirely! (Keep building & publishing it, but no need to version or store it here at all). Not critical for this PR, but lets look at it as part of v2.

Reply:
Yeah, I assumed this was your intention, based on how that directory has been managed in the past. I updated the `dist` folder here with that in mind, actually: this is not going into the `main` branch (so it doesn't interrupt anyone using GitHub’s default branch as a source to load from) but having a build available allows people to more easily test and give feedback on on all the breaking changes and philosophical changes landing in v2. But I can pull it back out if you’d prefer.

Reply:
Happy to leave it for now for easy testing, but I think it would be good to drop `dist` before the final v2 release.